backend/.env.example

```
@@ -16,6 +16,16 @@ JWT_EXPIRE=7d
 # CORS Configuration
 CORS_ORIGIN=http://localhost:5173
 
+# Frontend URL (used for OAuth redirects)
+FRONTEND_URL=http://localhost:5173
+
+# Google OAuth
+GOOGLE_CLIENT_ID=your-google-client-id.apps.googleusercontent.com
+GOOGLE_CLIENT_SECRET=your-google-client-secret
+GOOGLE_REDIRECT_URI=http://localhost:5000/api/auth/google/callback
+
+Guide :https://github.com/ankurdotio/Difference-Backend-video/tree/main/025-googleoauth
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The Guide :https://github.com/ankurdotio/Difference-Backend-video/tree/main/025-googleoauth key has incorrect delimiter

(IncorrectDelimiter)